src/components/InfiniteList/InfiniteList.tsx

```diff
@@ -103,15 +105,10 @@ export const InfiniteList = <T, C>({
       Footer: loading ? Loading : undefined,
       EmptyPlaceholder: loading ? undefined : () => EmptyPlaceholder,
       List: ListContainer,
-      Item,
+      Item: ItemOverride ?? Item,
       ScrollSeekPlaceholder: SkeletonItem,
       ...props.components,
     },
-    scrollSeekConfiguration: {
```

Comment:
This was originally added [here](https://github.com/CruGlobal/mpdx-react/commit/69c74d8c786b0f3b6dad061a9f866f2c09fd56de#diff-ddcb4d4a5c2069166252f62cd5c7243f3c6b1e1e8f3e9bac5bdab0831b895404), but no reason was given as to why.

Reply:
I've come across this a couple times and never had the guts to just remove it. Thanks for doing that! I think it makes things seem a lot more janky than it helps.